int64[] int_list
---
string res
